sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,7 +9,8 @@
 
 #### Bugs Fixed
 * Fixed a bug where change feed query in Async client was not returning all pages due to case-sensitive response headers. See [PR 35090](https://github.com/Azure/azure-sdk-for-python/pull/35090)
-* Fixed a bug when a retryable exception occurs in the first page of a query execution causing query to return 0 results. See [PR 35090](https://github.com/Azure/azure-sdk-for-python/pull/35090).
+* Fixed a bug when a retryable exception occurs in the first page of a query execution causing query to return 0 results. See [PR 35090](https://github.com/Azure/azure-sdk-for-python/pull/35090)
+* Fixed a typo that caused the link to the concurrency sample to be broken in README.MD. See [PR35248](https://github.com/Azure/azure-sdk-for-python/pull/35248)
```

Comment:
no need for a changelog entry to an update to the readme - there's no code changes